apps/studio/components/interfaces/App/FeaturePreview/FeaturePreviewContext.tsx

```diff
@@ -10,6 +10,13 @@ import { SqlEditorTabsPreview } from './SqlEditorTabs'
 import { TableEditorTabsPreview } from './TableEditorTabs'
 
 export const FEATURE_PREVIEWS = [
+  {
+    key: LOCAL_STORAGE_KEYS.UI_NEW_LAYOUT_PREVIEW,
+    name: 'Dashboard layout update',
+    content: <APISidePanelPreview />,
```

Comment:
This is showing the API Side panel preview and not the new dashboard layout update in the feature preview

Reply:
added a new component for the layout preview 🙏